app/views/account/companies/show.html.slim

```diff
@@ -67,3 +67,18 @@
           .card-content
             .card-body
               = render 'account/companies/html/social_links', company: @company
+        - unless @twitter.tweets.blank?

```

Comment:
if tweets.any?

try to avoid using unless, in most cases you can use "if" and it's more readable